### PR TITLE
Support COPY statement dependencies in Assembler

### PR DESCRIPTION
**Pull request by drbruce-git**

### CODE REVIEW

File: samples/application-conf/application.properties [hunk 1]
```diff
@@ -66,15 +66,22 @@ plincRule = {"library": "SYSLIB", \
                  ] \
                }
 
-# Rule to locate ASM macros. This rule defaults to the local maclib folder
-# in the main application folder.
```
Comment (dennis-behm) [inline samples/application-conf/application.properties:70]: Can you please bring back the description of the maclibRule?

File: samples/application-conf/application.properties [hunk 2]
```diff
@@ -66,15 +66,22 @@ plincRule = {"library": "SYSLIB", \
...
+
 # Category filters on what you want to include during the scanner (i.e. excludes macro-def keyword)
 maclibRule = {"library": "SYSLIB", "category": "MACRO", \
                   "searchPath": [ \
                     {"sourceDir": "${workspace}", "directory": "${application}/maclib"} \
                  ] \
                }
 
+# Rule to locate ASM mcopy books. This rule defaults to the local maclib folder
```
Comment (dennis-behm) [inline samples/application-conf/application.properties:77]: Typo: 
```
Rule to locate ASM mcopy books
```
should be 
```
Rule to locate ASM copybooks
```

**Review by dennis-behm — COMMENTED** [review]
@drbruce-git Can you please check on the comments? Thanks!

File: samples/application-conf/application.properties (hunk 2 above)
Comment (drbruce-git) [reply samples/application-conf/application.properties]: Fixes

**Pull request merged**